src/components/bucket/create/create.html

```diff
@@ -0,0 +1,64 @@
+<md-dialog class="create-dialog" aria-label="Create Bucket Dialog">
```

Comment:
The class name `create-dialog` would be better to rename to a normal class name for reusable.